float64 x
float64 y
float64 z
float64 wx
float64 wy
float64 wz
bool reset_rendering
---
float64 fx
float64 fy
float64 cx
float64 cy
bool result
